README.md

```diff
@@ -116,6 +116,8 @@ In a few moments you will have Kubernetes up and running on your Raspberry Pi 2,
 
 * Flash an SD card using [Raspbian Lite](https://www.raspberrypi.org/downloads/raspbian/)
 
+* In order to enable SSH, create an empty file named `ssh` on the `boot` volume: `touch /Volumes/boot/ssh`
```

Comment:
I would not assume that all users are on MacOS. I know from the download stats that they aren't.  Probably just say create a `ssh` file in the boot partition?

Reply:
Makes sense. I will update 👍